11ty/definitions/neo-colonialism.md

```diff
@@ -0,0 +1,25 @@
+---
+title: neo-colonialism
+slug: neo-colonialism
+speech: noun
+excerpt: the continued influence of colonial era ideologies and practices in a country or region by former dependencies, generally enforced by its own government and/or citizens as a lingering effect of colonialism
+sub_terms:
+    - text: neo-
+    full_title: neo-colonialism
+    - text: modern
+    full_title: modern colonialism
```

Comment:
```suggestion
skip_in_table_of_content: true
```
Because this term is itself a sub-term, we will use this line of front-matter in order to make it skipped in the table of contents.